src/components/layout/Sidebar/Sidebar.scss

```diff
@@ -6,7 +6,8 @@
   left: 50%;
   transform: translateX(-50%);
   gap: 1em;
-  height: 3em;
+  width: 60px;
```

Comment:
I'm personally a big fan of relative units, why go with `px` here

Reply:
Definitely, lets keep using `rem` here.